game/scripts/npc/items/item_armlet.txt

```diff
@@ -86,22 +86,22 @@
       "07"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "unholy_bonus_strength"                           "25 55 85 110 140"
+        "unholy_bonus_strength"                           "25 50 80 110 140"
       }
       "08"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "unholy_bonus_armor"                              "4 7 8 9 10"
+        "unholy_bonus_armor"                              "4 6 8 10 12"
       }
       "09"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "unholy_health_drain_per_tick"                    "6 8 11 14 18"
+        "unholy_health_drain_per_tick"                    "6 9 12 15 19"
```

Comment:
I still think this should be: bonus_health_regen (5 10 15 20 25) + 1 to avoid Huskar shenanigans. So I would put `6 11 16 21 26`. If you think its too much at levels 4 and 5 then: `6 11 16 18 20'

Reply:
Level 2 armlet can't be 11 because then its back to excactly what is was before. Being unpurchaseable on anyone other then Huskar because the item drains too much HP. Level 3+ Armlet can be nerfed.

Reply:
I reduced the HP regen on level 2 armlet by 1. I increased the hearlth drain on level 3+ Armlet to "13 18 22" and increased the health_regen to "15 21 27"